examples/bert/pytorch_job_bert.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: "kubeflow.org/v1"
+kind: "PyTorchJob"
+metadata:
+  name: "pytorch-job-bert"
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      imagePullPolicy: Always
+      restartPolicy: OnFailure
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          containers:
+            - name: pytorch
+              image: <username>/pytorch_dist_bert:latest
```

Comment:
em. This is ok at this moment.  We should host some example images somewhere in the future.